src/common/modules/data/DefaultSettings.js

```diff
@@ -18,9 +17,7 @@ const defaultSettings = {
     },
     emojiPicker: {
         set: "native",
-        native: true,
```

Comment:
Write settings migration to delete this!

Reply:
Okay and tested manually:
![image](https://github.com/user-attachments/assets/8604b151-c520-49b9-b649-31e0f85c39a2)